pages/projects/virtual-concierge/index.js

```diff
@@ -0,0 +1,129 @@
+          </h2>
+          <div className="grid lg:grid-cols-2 gap-6  ">
+            <div className=" bg-gradient-to-tr from-custom-blue-blue to-vc-blue-lt  p-4 ">
+              <img
+                className=" object-fill w-80 h-auto mx-auto"
+                src="/virtualconcierge/VirtualConcierge.svg"
+                alt="Virtual Assistant Logo"
+              />
+            </div>
+
+            <p className="mb-4 px-1 lg:px-0 ">{t("vc:virtualAssistantBio")}</p>
+          </div>
+        </div>
+        {/* the scenario section */}
+        <div>
+          <h1
+            className="mb-6 mt-8 text-h1"
+            tabIndex="-1"
+            id="virtualAssistantTitle"
+          >
+            {t("vc:sectionTitle")}
+          </h1>
+          <VirtualConcierge
+            dataTestId="scenario1"
+            dataCy="scenario1"
+            description={t("vc:scenarioPart1")}
+            description1={t("vc:scenarioPart1-1")}
+            className=" bg-gray-200"
+            vcImage={
+              props.locale === "fr"
+                ? "/virtualconcierge/fr/virtual-concierge1.png"
+                : "/virtualconcierge/en/virtual-concierge1.png"
+            }
+            vcImageAltText={t("vc:altText1")}
+          />
+
+          <VirtualConcierge
+            dataTestId="scenario2"
+            dataCy="scenario2"
+            description={t("vc:scenarioPart2")}
+            className=" bg-gray-200"
+            vcImage={
+              props.locale === "fr"
+                ? "/virtualconcierge/fr/virtual-concierge2.png"
+                : "/virtualconcierge/en/virtual-concierge2.png"
+            }
+            vcImageAltText={t("vc:altText2")}
+          />
+          <VirtualConcierge
+            dataTestId="scenario3"
+            dataCy="scenario3"
+            description={t("vc:scenarioPart3")}
+            className=" bg-gray-200"
+            vcImage={
+              props.locale === "fr"
+                ? "/virtualconcierge/fr/virtual-concierge3.png"
+                : "/virtualconcierge/en/virtual-concierge3.png"
+            }
+            vcImageAltText={t("vc:altText3")}
+          />
+          <p className="pb-2 ">{t("vc:scenarioPart4")}</p>
+          <p className="pb-2 ">{t("vc:scenarioPart4-1")}</p>
+          <p className="pb-2 mb-6">{t("vc:scenarioPart4-2")}</p>
+        </div>
+
+        {/*TODO these are the buttons on the bottom of the page that can be added if needed.  */}
+        {/* <div className="flex-col flex lg:flex lg:flex-row gap-1">
+          <TextButtonField href="/experiments" buttonText={t("experimentsBtnTxt")} idButton="ExperimentsButton" dataCyButton="ExperimentsButton" className="flex" />
+          <TextButtonField href="/about" buttonText={t("learnMoreAboutSCL")} idButton="AboutButton" dataCyButton="AboutButton" className="flex" />
+        </div> */}
+      </section>
+      <CallToAction
+        title={t("signupTitle")}
+        description={t("becomeAParticipantDescription")}
```

Comment:
The `description` should be changed to `html` to render the unsubscribe link.
![image](https://user-images.githubusercontent.com/77116011/126331225-25c7cc80-46a1-4849-be49-5b0af54eed24.png)

Reply:
changed description to html